eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -184,6 +184,10 @@ jobs:
             ServiceDirectory: ${{ parameters.ServiceDirectory }}
             TestPipeline: ${{ parameters.TestPipeline }}
 
+      - pwsh: |
+          Write-Host "##vso[task.setvariable variable=DOTNET_ROLL_FORWARD]Major"
```

Comment:
@gearama in case you missed this:
https://github.com/Azure/azure-sdk-for-cpp/pull/5681#discussion_r1638525904

> With [Azure/azure-sdk-tools#8373](https://github.com/Azure/azure-sdk-tools/pull/8373) and the sync #5699 merged, do we still need this change?
> 
> It seems like the default is "Major", now.

Reply:
yes, i discussed it with Daniel separatelly, the basic idea is Scott hardcoded the proxy version that we use in our repo, in order to use the latest version of the proxy that has the framework built in ( to avoid using externaal dep) i have to make update / test the new proxy version. I would rather do it separatelly from this work to not make this PR into a mega convolutes PR with all the changes packed in

Reply:
Ah, that make sense! Thanks